### PR TITLE
EASYOPAC-222 - Update module, to use new ting_covers to get item images.

### PR DESCRIPTION
**Pull request by makcumm**

### CODE REVIEW

File: templates/alma-item-list_item.tpl.php [hunk 1]
```diff
@@ -5,10 +5,8 @@
  * Template file for single item list item.
  */
 ?>
-<div class="alma-item-list-item item-local_id-<?php print $local_id; ?>">
-  <div class="ting-cover ting-cover-processing" data-local_id="<?php print $local_id; ?>">
-    <a href="/ting/object/<?php print $faust; ?>"></a>
-  </div>
+<div class="alma-item-list-item item-local_id-<?php print $faust; ?>">
+  <?php print $image; ?>
```
Comment (guddo) [inline templates/alma-item-list_item.tpl.php:9]: Why did you remove the "processing" `<div>` and the link `<a>`?

File: alma_item_list.module [hunk 2]
```diff
@@ -159,6 +151,15 @@ function alma_item_list_get_content($query, $limit, $cache) {
         'ajax'
       );
 
+      $img = array(
+        '#theme' => 'ting_object_cover',
+        '#object' => $ding_entity,
+        '#image_style' => 'ding_item_list3',
```
Comment (guddo) [inline alma_item_list.module]: Is this a legit image style?

File: templates/alma-item-list_item.tpl.php (hunk 1 above)
Comment (makcumm) [reply templates/alma-item-list_item.tpl.php:9]: As you can see here https://github.com/easySuite/alma_item_list/pull/3/files#diff-2008b07ce66bef30a72c4aa4b479d262R154
I used renderable array, which use "ting_object_cover" theme function, which will render image.
